libethash-cuda/CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ else()
 		"-gencode arch=compute_60,code=sm_60"
 		"-gencode arch=compute_61,code=sm_61"
 		"-gencode arch=compute_62,code=sm_62"
-		"-gencode arch=compute_70,code=sm_70"
```

Comment:
?

Reply:
@chfast Not usable hardware. I guess that pretty well settles it. Cuda 9 only support.

Reply:
And yes, we keep sm 70. That was just to make cuda 8 work